Services/Facades/PlacedBetFacade.cs

```diff
@@ -105,8 +110,17 @@ public async Task<PlacedBetsDto> UpdatePlacedBetByIdAsync(Guid id, UpdatePlacedB
                 return null;
             }
 
-            PlacedBetsDto updatedPlacedBetDto = await _placedBetsService.UpdateByIdAsync(id, updatePlacedBet);
-            return updatedPlacedBetDto;
+            using var transaction = _dbContext.Database.BeginTransaction();
+            try
+            {
+                PlacedBetsDto updatedPlacedBetDto = await _placedBetsService.UpdateByIdAsync(id, updatePlacedBet);
+                return updatedPlacedBetDto;
+            }
+            catch (Exception ex)
+            {
+                transaction.Rollback();
+                throw ex;
+            }
```

Comment:
I don't think you need to use try catch. the transaction will be rollbacked if an error occurs. 

you need to commit the transaction when the operation is done